doc/lua_api.txt

```diff
@@ -5400,7 +5399,14 @@ This is basically a reference to a C++ `ServerActiveObject`
 
 * `get_pos()`: returns `{x=num, y=num, z=num}`
 * `set_pos(pos)`: `pos`=`{x=num, y=num, z=num}`
-* `move_to(pos, continuous=false)`: interpolated move
+* `move_to(pos, continuous=false)`
+    * Does an interpolated move for lua entities. The object will not instantly
+      be at `pos` but can be watched how it gets there.
+    * If the objectref is a lua entity and `continuous` is true, object will not
+      be moved to the current position before starting the interpolated move.
+    * For players this is the same as `get_pos` with the only difference
+      that the mapblock in which the player ends up is not force-sent.
```

Comment:
Force sending is an internal gameplay optimization, and irrelevant to modders or players. Also refer to `ObjectRef` (CamelCase, including code formatting) above, please.

Reply:
I wonder if it was good to document the automatic block sending for `set_pos` at `send_mapblock`. :thinking: 
https://github.com/minetest/minetest/blob/6b489b64616b0f34c1a36474fdfe3feb90ac6800/doc/lua_api.txt#L5651-L5655